hledger/test/balance/multicommodity.test

```diff
@@ -1,31 +1,60 @@
-# 1. In tabular balance reports, cap the maximum width and elide if sufficiently large
 <
 2020-01-01
   (a)  1.00A
   (a)  1.00B
   (a)  1.00C
   (a)  1.00D
+
+2021-01-01
+  (a)  1.00D
   (a)  1.00E
   (a)  1.00F
 
+# 1. In tabular balance reports, display on a single line by default
 $ hledger -f- bal -Y
-Balance changes in 2020:
+Balance changes in 2020-01-01..2021-12-31:
 
-   ||                          2020 
-===++===============================
- a || 1.00A, 1.00B, 1.00C, 3 more.. 
----++-------------------------------
-   || 1.00A, 1.00B, 1.00C, 3 more.. 
+   ||                       2020                 2021 
+===++=================================================
+ a || 1.00A, 1.00B, 1.00C, 1.00D  1.00D, 1.00E, 1.00F 
+---++-------------------------------------------------
+   || 1.00A, 1.00B, 1.00C, 1.00D  1.00D, 1.00E, 1.00F 
 
-# 2. Unless --no-elide is used.
-$ hledger -f- bal -Y --no-elide
-Balance changes in 2020:
+# 2. Display multiline if requested
+$ hledger -f- bal -Y --commodity-layout=multiline
+Balance changes in 2020-01-01..2021-12-31:
+
+   ||  2020   2021 
+===++==============
+ a || 1.00A  1.00D 
+ a || 1.00B  1.00E 
+ a || 1.00C  1.00F 
+ a || 1.00D      0 
```

Comment:
I have doubts about this 0. Usually a number in a report means something quantitative, but unlike `--commodity-layout=column`, here this cell has no real meaning, and the 0 is simply cosmetic to avoid empty spaces. I still need to try it out in practice.

Reply:
Yes, happy to either have a space or 0. Your call on this one.

Reply:
I'm inclined to show a space right now. It helps differentiate `multiline` and `column` modes. It would be harder for machine parsing but `multiline` isn't intended for that.